src/shared/components/paginationControls/PaginationControls.tsx

```diff
@@ -41,6 +41,7 @@ export interface IPaginationControlsProps {
     lastPageDisabled?:boolean;
     pageNumberEditable?:boolean;
     groupButtons?:boolean;
+    hidePaginationIfOnePage?:boolean;
```

Comment:
i don't think this needs to be an option

Reply:
really? I could see an aesthetic use case for it